Example_Programs/programming_concpets/calculator_complex/test/test_complex_calculator.c

```diff
@@ -25,6 +27,18 @@ void test_zero(void)
     TEST_ASSERT_EQUAL(SUCCESS, complex_sum(&c1, &c2, &result));
     TEST_ASSERT_EQUAL(0, result.real);
     TEST_ASSERT_EQUAL(0, result.imaginary);
+

```

Comment:
Write different test cases for each function. It is more readable and also easy to find the bugs.

Reply:
Sure sir I will do it.